numpy/ma/tests/test_core.py

```diff
@@ -1868,16 +1868,25 @@ def test_extremum_fill_value(self):
         assert_equal(test['B']['BB'], default_fill_value(a['B']['BB']))
 
         test = minimum_fill_value(a)
-        assert_equal(test[0], minimum_fill_value(a['A']))
-        assert_equal(test[1][0], minimum_fill_value(a['B']['BA']))
-        assert_equal(test[1][1], minimum_fill_value(a['B']['BB']))
-        assert_equal(test[1], minimum_fill_value(a['B']))
+        assert_equal(test['A'], minimum_fill_value(a['A']))
+        assert_equal(test['B']['BA'], minimum_fill_value(a['B']['BA']))
+        assert_equal(test['B']['BB'], minimum_fill_value(a['B']['BB']))
+        assert_equal(test['B'], minimum_fill_value(a['B']))
```

Comment:
Worst case, we can coerce the `0d array` back to tuple in the final step, but that's probably less desirable.

`default_fill_value` already returned a 0d array in this situation